02Lesson/02Project/index.html

```diff
@@ -1,10 +1,51 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Daily Planet</title>
   <link rel="stylesheet" href="css/style.css"/>
+  <meta charset="UTF-8">
 </head>
 <body>
-  <!-- You code start here -->
+  <header>
+    <h1>Daily Planet</h1>
+    <nav>
+      <a href="">Sports</a>
+      <a href="">Local</a>
+      <a href="">Economy</a>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <article>
+        <h2>Able an hope of body</h2>
+        <hr>
+        <p>
+          Bringing unlocked me an striking ye perceive. Mr by wound hours oh happy. Me in resolution pianoforte continuing we. Most my no spot felt by no. He he in forfeited furniture sweetness he arranging. Me tedious so to behaved written account ferrars moments. Too objection for elsewhere her preferred allowance her. Marianne shutters mr steepest to me. Up mr ignorant produced distance although is sociable blessing. Ham whom call all lain like.
+        </p>
+        <p>
+          Breakfast agreeable incommode departure it an. By ignorant at on wondered relation. Enough at tastes really so cousin am of. Extensive therefore supported by extremity of contented. Is pursuit compact demesne invited elderly be. View him she roof tell her case has sigh. Moreover is possible he admitted sociable concerns. By in cold no less been sent hard hill.
+        </p>
+        <img src="http://www.publicdomainpictures.net/pictures/20000/velka/maple-tree-in-park-110661300191011Zru.jpg" width="650" alt="some image"/>
+        <p>
+          Seen you eyes son show. Far two unaffected one alteration apartments celebrated but middletons interested. Described deficient applauded consisted my me do. Passed edward two talent effect seemed engage six. On ye great do child sorry lived. Proceed cottage far letters ashamed get clothes day. Stairs regret at if matter to. On as needed almost at basket remain. By improved sensible servants children striking in surprise.
+        </p>
+      </article>
+    </section>
+    <aside>
+      <h3>Sponsors</h3>
+      <ul>
+        <li>Sponsor 1</li>
+        <li>Sponsor 2</li>
+      </ul>
+      <h3>Other Stuff</h3>
+      <ul>
+        <li>Stuff</li>
+        <li>Stuff</li>
+      </ul>
+    </aside>
+  </main>
+  <footer>
+    <h6>The Daily Planet 2016 Copyright</h6>

```

Comment:
I wouldn't use headline tags in the footer, they are meant for headlines 😄  If you just need a tag to style text, use `span`